app/code/community/Bolt/Boltpay/etc/config.xml

```diff
@@ -176,13 +176,6 @@
 
   <admin>
     <routers>
```

Comment:
can you send me separate PR for this? assuming it's not related to product page checkout

Reply:
> LGTM, can you attach screenshot of success screen?

See the screenshot here https://screencast.com/t/yQvvWj7es